### PR TITLE
test(grid): [grid]add grid header related unit test

### PR DESCRIPTION
**Pull request by AcWrong02**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/vue/src/grid/__tests__/grid.test.tsx [hunk 1]
```diff
@@ -71,7 +140,7 @@ describe('PC Mode', () => {
   })
 
   // slots
-  test('default-slot', async () => {
+  test.skip('default-slot', async () => {
```
Comment (kagol) [inline packages/vue/src/grid/__tests__/grid.test.tsx:143]: @AcWrong02 为什么跳过了这些用例呢？

**Comment by kagol** [comment]
@AcWrong02 建议优先补充组件的E2E测试用例，而不是单元测试用例，E2E测试用例有加入PR门禁中，在PR标题中加入 `[grid]`，比如：`refactor(grid): [grid] refactor the basic feature of TinyGrid Component`。

可参考贡献指南文档：[触发组件的 e2e 测试](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.zh-CN.md#pull-request-%E7%9A%84%E6%A0%87%E9%A2%98)

File: packages/vue/src/grid/__tests__/grid.test.tsx (hunk 1 above)
Comment (AcWrong02) [reply packages/vue/src/grid/__tests__/grid.test.tsx:143]: 在第一个pr中有编写，后面还需要进行整合

**Review by Zcating — CHANGES_REQUESTED** [review]
把两处代码先回滚一下吧，不用skip.

**Pull request closed without merging**